projects/grafana/grafana-ocp.yaml

```diff
@@ -0,0 +1,717 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: grafana-ocp
+  annotations:
+    "openshift.io/display-name": Grafana ocp
+    description: |
+      Grafana server with patched Prometheus datasource.
+    iconClass: icon-cogs
+    tags: "metrics,monitoring,grafana,prometheus"
+parameters:
+- description: The location of the proxy image
+  name: IMAGE_PROXY
+  value: openshift/oauth-proxy:v1.0.0
+- description: External URL for the grafana route
+  name: ROUTE_URL
+  value: ""
+- description: The namespace to instantiate heapster under. Defaults to 'grafana'.
+  name: NAMESPACE
+  value: grafana
+- description: The session secret for the proxy
+  name: SESSION_SECRET
+  generate: expression
+  from: "[a-zA-Z0-9]{43}"
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: grafana-ocp
+    namespace: "${NAMESPACE}"
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirectreference.primary: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"grafana-ocp"}}'
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: gf-cluster-reader
+  roleRef:
+    name: cluster-reader
```

Comment:
I believe the permissions we had Grafana run with before was `view` in the namespace that Prometheus is deployed in. Why do we need this?

Reply:
will give a try without this, and update accordingly

Reply:
This is still outstanding?

Reply:
right, just forgot to remove it. 10x